setup.py

```diff
@@ -86,38 +86,38 @@
     'pillow>=3.4.2,<4',
     'Babel==2.5.3',
     'cryptography==1.7.1',
-    'Django>=1.8,<1.10',
-    'django-bootstrap3>=6.1,<7',
-    'django-countries>=3.3,<4',
-    'django-enumfields>=0.7.2,<0.8',
-    'django-filer>=1.0,<2',
-    'django-filter>=1.0,<1.1',
-    'django-jinja>=1.4,<2',
-    'django-mptt>=0.8.0,<0.9',
-    'django-parler>=1.5,<1.6.3',
-    'django-parler-rest>=1.3a1,<2',
-    'django-polymorphic>=0.8.0,<0.10',
-    'django-registration-redux>=1.2,<1.5',
-    'django-timezone-field>=1.2,<2',
+    'Django>=1.8,<2',
+    'django-bootstrap3>=9,<10',
+    'django-countries>=5,<5.3',
+    'django-enumfields>=0.9,<0.10',
+    'django-filer>=1.3,<2',
+    'django-filter>=1.1,<1.2',
+    'django-jinja>=2.3,<3',
+    'django-mptt>=0.9,<0.10',
+    'django-parler>=1.9,<2',
+    'django-parler-rest>=1.4.2,<2',
+    'django-polymorphic>=0.9.2,<2.1.0',  # For Django 1.8 and 1.9 use version lower 1
+    'django-registration-redux>=1.5,<1.9',
+    'django-timezone-field>=2.1,<3',
```

Comment:
Are all these requirement upgrades really needed?

I'd prefer not to change to lowest required version unless they're really a must. It's not Shuup's concern if all those required libraries work together. Shuup should only tell which are the lowest (and highest) versions it needs and then the project installing Shuup should take care that compatible versions are installed. (Usually that is the newest available versions at that point.)